lib/node_modules/@stdlib/math/base/special/sincospi/test/test.js

```diff
@@ -27,35 +26,36 @@ tape( 'main export is a function', function test( t ) {
 	t.end();
 });
 
-tape( 'if provided positive infinity, the function returns `NaN`', function test( t ) {
-	var y = sincospi( PINF );
-	t.equal( isnan( y[0] ), true, 'returns NaN when provided NaN' );
-	t.equal( isnan( y[1] ), true, 'returns NaN when provided NaN' );
-	t.end();
-});
-
-tape( 'if provided negative infinity, the function returns `NaN`', function test( t ) {
+tape( 'if provided negative infinity, the function returns `[NaN,NaN]`', function test( t ) {
 	var y = sincospi( NINF );
-	t.equal( isnan( y[0] ), true, 'returns NaN when provided NaN' );
-	t.equal( isnan( y[1] ), true, 'returns NaN when provided NaN' );
+	t.equal( isnan( y[0] ), true, 'returns NaN when provided negative infinity' );
+	t.equal( isnan( y[1] ), true, 'returns NaN when provided negative infinity' );
 	t.end();
 });
 
-tape( 'the function throws a range error if provided either positive infinity', function test( t ) {
+tape( 'if provided positive infinity, the function returns `[NaN,NaN]`', function test( t ) {
 	var y = sincospi( PINF );
-	t.equal( isnan( y[0] ), true, 'returns NaN when provided NaN' );
-	t.equal( isnan( y[1] ), true, 'returns NaN when provided NaN' );
+	t.equal( isnan( y[0] ), true, 'returns NaN when provided positive infinity' );
+	t.equal( isnan( y[1] ), true, 'returns NaN when provided positive infinity' );
 	t.end();
 });
 
-tape( 'if provided `NaN`, the function returns `NaN`', function test( t ) {
+tape( 'if provided `NaN`, the function returns `[NaN,NaN]`', function test( t ) {
 	var y = sincospi( NaN );
 	t.equal( isnan( y[0] ), true, 'returns NaN when provided NaN' );
 	t.equal( isnan( y[1] ), true, 'returns NaN when provided NaN' );
 	t.end();
 });
 
-tape( 'if provided an integer, the function returns `+-1.0`', function test( t ) {
+tape( 'it computes results in-place and returns a reference to the input array', function test( t ) {
```

Comment:
Would change the "it" to "the function" in this test description..

Reply:
✅